<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpleShapeRectAngle</class>
 <widget class="QDialog" name="simpleShapeRectAngle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simpleShapeRectAngle</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="xLabelObject">
        <property name="text">
         <string>xPosition</string>
        </property>
        <property name="buddy">
         <cstring>xLineEditObject</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yLabelObject">
        <property name="text">
         <string>yPosition</string>
        </property>
        <property name="buddy">
         <cstring>yLineEditObject</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="widthLabelObject">
        <property name="text">
         <string>width</string>
        </property>
        <property name="buddy">
         <cstring>widthLineEditObject</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="heightLabelObject">
        <property name="text">
         <string>height</string>
        </property>
        <property name="buddy">
         <cstring>heightLineEditObject</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="xLineEditObject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="yLineEditObject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="widthLineEditObject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="heightLineEditObject"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xLineEditObject</tabstop>
  <tabstop>yLineEditObject</tabstop>
  <tabstop>widthLineEditObject</tabstop>
  <tabstop>heightLineEditObject</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>simpleShapeRectAngle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>simpleShapeRectAngle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
